tests/experimental_combination/integration_testing/settings.py

```diff
@@ -3,19 +3,18 @@
 
 import os
 
-PROJECT = os.environ.get('GOOGLE_CLOUD_PROJECT', 'cdot-rtdh-dev')
+PROJECT = os.environ.get("GOOGLE_CLOUD_PROJECT", "cdot-rtdh-dev")
```

Comment:
Should we remove these defaults from the public repo?

Reply:
removed this whole directory, it is from the rtdh repos, not useful here